optimism/src/lookups.rs

```diff
@@ -127,32 +67,17 @@ pub trait Lookups {
     fn lookups(&mut self);
 }
 
-/// A table of values that can be used for a lookup, along with the ID for the table.
-#[derive(Debug, Clone)]
-pub struct LookupTable<F> {
-    /// Table ID corresponding to this table
-    #[allow(dead_code)]
-    table_id: LookupTableIDs,
-    /// Vector of values inside each entry of the table
-    #[allow(dead_code)]
-    entries: Vec<Vec<F>>,
+/// Trait that creates all the fixed lookup tables used in the VM
+pub(crate) trait FixedLookupTables<F> {
```

Comment:
Subjective: If the trait is not going to have more than one instance, one can just make these functions top-level citizens and put them in a separate file, used like `concrete_tables::table_reset()` and so on.

Reply:
Oh, they will probably contain more functions as we start using them in the protocol and realize we need extra functionalities 😃

Reply:
Nothing wrong with many top-level functions :shrug: If there are many of them it's probably good not to duplicate it twice